engine/src/flutter/lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -153,6 +153,75 @@ abstract class PlatformDispatcher {
   double scaleFontSize(double unscaledFontSize);
 }
 
+final class SystemColor {
```

Comment:
Same thing here, do we or should we have mechanism to get notification when system color change during the life time of an app?

Reply:
I'm not yet sure how to implement it. The only signal I can listen to is the `forced-colors` media query, but that doesn't notify about individual color changes, only about the high contrast switch overall. I think it might be fine for now though. I expect users will create a theme once, and then turn the high contrast mode on and off, which already send a notification.